### DISCUSSION TITLE
Likely ETA of next release

### DISCUSSION THREAD

**Discussion by munckymagik** [valhalla/valhalla#4582] [General]
Hi. 

Not intending to put pressure on, just asking so we can plan whether to target master to pick up [this fix (thank you)](https://github.com/valhalla/valhalla/pull/4547) or whether it's due soon and we should wait?

Cheers

**Comment by nilsnolde** [comment]
Personally I’d like to release asap. I think we’re quite good with closed tasks now (lots of more good stuff almost mergeable), maybe we’d want to include the final landmarks PR, in which case it’d take another 2-3 weeks minimum. 

The changes are quite a lot since last release, we kinda dropped the ball there again for a while..

**Comment by azarz** [comment]
Hello,

I have a similar question, as I'd like to include [this feature](https://github.com/valhalla/valhalla/pull/4524) in the next release of my service. No pressure, I just want the info! 😉

**Reply by chrstnbwnkl** [reply]
Valhalla hasn't been following a strict release pattern for some time. The tagged releases are generally not guaranteed to be any better/more stable than any other non-tagged commit. If you want to use a specific feature that was recently added, I would advise you to have your own tests tailored to your specific use case and just try out the commit that added that feature.

**Reply by azarz** [reply]
Is it OK if I use the docker image like that then?
```Dockerfile
FROM ghcr.io/valhalla/valhalla@sha256:6fb108a960bdc63c12b1c92ab04517a0c56a019ef45c9b9c8cf847d8de9db72b
```
(I got the sha256 from the last github action run on master)
